src/app/(auth)/signin.tsx

```diff
@@ -27,74 +44,143 @@ import { resetNavigationStack } from "@/src/lib/resetNavigationStack";
-        />
+    <View className="flex-1 items-center pt-14">
+      <View className="w-11/12 flex-row items-center justify-between pb-5">
+        <View className="flex-row items-center">
+          {/* <Image
+            className="h-6 w-6 rounded-[20px]"
+            source={require("../../../assets/images/Logo.svg")}
+          /> */}
+          <Logo height={24} width={24} />
+          <Text className="ml-2 text-2xl font-bold">Habit Together</Text>
+        </View>
+        <Pressable className="rounded-2xl bg-stone-200 px-4 py-1 dark:border-2 dark:border-stone-300 dark:bg-stone-800">
+          <Text className="text-base font-semibold">Skip</Text>
+        </Pressable>
       </View>
-      <View style={styles.verticallySpaced}>
-        <Input
-          label="Password"
-          leftIcon={{ type: "font-awesome", name: "lock" }}
-          onChangeText={(text) => setPassword(text)}
-          value={password}
-          secureTextEntry={true}
-          // placeholder="password"
-          autoCapitalize={"none"}
+
+      <FlatList
+        data={tutorialData}
+        renderItem={({ item }) => TutorialItem({ item })}
+        horizontal
+        pagingEnabled
+        showsHorizontalScrollIndicator={false}
+        keyExtractor={(item) => item.pageNum}
+        onScroll={Animated.event(
+          [{ nativeEvent: { contentOffset: { x: scrollX } } }],
+          {
+            useNativeDriver: false,
+          },
+        )}
+        className="mb-14 grow-0"
+      />
+
+      <SlidingDot
+        data={tutorialData}
+        scrollX={scrollX}
+        dotSize={10}
+        containerStyle={{ position: "relative" }}
+        dotStyle={{
+          backgroundColor: colorScheme === "dark" ? colors.white : colors.black,
+        }}
+        marginHorizontal={10}
+        slidingIndicatorStyle={{
+          backgroundColor: colorScheme === "dark" ? colors.white : colors.black,
+        }}
+      />
+
+      <Text className="mb-3 w-2/3 text-center text-xs text-stone-400">
+        By continuing, I agree to the {"\n"}
+        <Link href="/">
+          <Text className="text-xs text-stone-400 underline">
+            Terms & Conditions
+          </Text>
+        </Link>{" "}
+        and{" "}
+        <Link href="/">
+          <Text className="text-xs text-stone-400 underline">
+            Privacy Policy
+          </Text>
+        </Link>
+      </Text>
+
+      {Platform.OS === "ios" && (
+        <>
+          <AuthButton
+            icon={<IconBrandAppleFilled fill={colorScheme === "dark" ? colors.white : colors.black} color={colorScheme === "dark" ? colors.white : colors.black} strokeWidth={0} size={24} />}
+            text="Continue with Apple"
+            onPress={() => { }}
+          />
+          <AuthButton
+            icon={<Image className="h-5 w-5 rounded-[20px]" source={require("../../../assets/images/google.png")} />}
+            text="Continue with Google"
+            onPress={() => { }}
+          />
```

Comment:
Google icon is pixelated. Download at a higher resolution (on Figma you can export at 4x).

Also create new components for buttons like Apple button and Google button so that you aren't copy pasting code

Reply:
It's not a big deal for Apple since you only use it once. But definitely for Google one (can just create a new component in this file)